scene/3d/lightmap_gi.cpp

```diff
@@ -1602,8 +1602,8 @@ Ref<CameraAttributes> LightmapGI::get_camera_attributes() const {
 PackedStringArray LightmapGI::get_configuration_warnings() const {
 	PackedStringArray warnings = VisualInstance3D::get_configuration_warnings();
 
-	if (OS::get_singleton()->get_current_rendering_method() == "gl_compatibility") {
-		warnings.push_back(RTR("Lightmap can only be baked from a device that supports the RD backends. Lightmap baking may fail."));
+	if (!DisplayServer::get_singleton()->can_create_rendering_device()) {
+		warnings.push_back(vformat(RTR("Lightmaps can only be baked from a GPU that supports the RenderingDevice backends.\nThis is likely because your GPU (%s) does not support Vulkan, Direct3D 12 or Metal. \nLightmap baking will not be available on this device, even though rendering existing baked lightmaps will work."), RenderingServer::get_singleton()->get_video_adapter_name()));
```

Comment:
```suggestion
		warnings.push_back(vformat(RTR("Lightmaps can only be baked from a GPU that supports the RenderingDevice backends.\nThis is likely because your GPU (%s) does not support Vulkan, Direct3D 12, or Metal. \nLightmap baking will not be available on this device, even though rendering existing baked lightmaps will work."), RenderingServer::get_singleton()->get_video_adapter_name()));
```
This sentence is missing something as the second part seems to imply that the lack of support is *why* it can only be baked from a GPU supporting a rendering device, unsure how to make it work